src/server/req_jobobit.c

```diff
@@ -1250,6 +1260,8 @@ on_job_rerun(struct work_task *ptask)
 					(preq->rq_reply.brp_code == DIS_EOD)) {
 					/* tcp connection to Mom broken */
 					conn_to_mom_failed(pjob, on_job_rerun);
+					free_br(preq);
+					NULL;
```

Comment:
preq = NULL;

Reply:
Oops - no idea how that happened! Thanks will fix asap